proxymode/proxymode.go

```diff
@@ -34,9 +33,9 @@ type ProxyOptions struct {
 }
 
 func NoProxy() *ProxyMode {
-	proxyMode, err := NewProxyMode(ProxyOptions{})
+	proxyMode, err := NewProxyMode(ProxyOptions{Enabled: false})
 	if err != nil {
-		log.Fatalf("no proxy mode should not return an error: %v", err)
+		panic(errors.Wrapf(err, "no proxy mode should not return an error"))
```

Comment:
Nit. Confusing double negation.
```suggestion
		panic(errors.Wrapf(err, "proxy mode should not return an error when disabled"))
```

Reply:
Rephrased